tox.ini

```diff
@@ -21,9 +20,10 @@ deps =
     django19: Django >= 1.9, < 1.10
     django110: Django >= 1.10, < 1.11
     django111: Django >= 1.11, < 1.12
-    django-dev: https://github.com/django/django/tarball/master
+    djangodev: https://github.com/django/django/tarball/master
 commands=
-    python runtests.py
+    coverage run --source polymorphic runtests.py
+    coverage xml -i
```

Comment:
I did not expect to have to do this for codecov, but I noticed that without this, codecov was printing the following message:
```
==> Detecting CI provider
    Travis Detected
==> Preparing upload
==> Processing gcov (disable by -X gcov)
    Executing gcov (find /home/travis/build/django-polymorphic/django-polymorphic -type f -name '*.gcno'  -exec gcov -pb  {} +)
==> Collecting reports
    Generating coverage xml reports for Python
    Error running `coverage xml -i`: None
    - Ignored: [Errno 2] No such file or directory: '/home/travis/build/django-polymorphic/django-polymorphic/coverage.xml'
Error: No coverage report found
```
After adding this line, the message changed...

```
==> Detecting CI provider
    Travis Detected
==> Preparing upload
==> Processing gcov (disable by -X gcov)
    Executing gcov (find /home/travis/build/django-polymorphic/django-polymorphic -type f -name '*.gcno'  -exec gcov -pb  {} +)
==> Collecting reports
    + /home/travis/build/django-polymorphic/django-polymorphic/coverage.xml bytes=95006
==> Appending environment variables
    + TRAVIS_PYTHON_NAME
    + TRAVIS_OS_NAME
==> Uploading
    .url https://codecov.io
    .query slug=django-polymorphic%2Fdjango-polymorphic&branch=master&package=py2.0.5&pr=277&job=221934631&commit=a57b067b425a40678238a88cf5cc330cc44b44aa&build=360.16&service=travis
    Pinging Codecov...
    Uploading to S3...
    https://codecov.io/github/django-polymorphic/django-polymorphic/commit/a57b067b425a40678238a88cf5cc330cc44b44aa
```

... and codecov made a comment on this issue.